Thrive.csproj

```diff
@@ -140,6 +140,13 @@
     <Compile Include="src\gui_common\DismissibleNotice.cs" />
     <Compile Include="src\gui_common\FocusFlowDynamicChildrenHelper.cs" />
     <Compile Include="src\gui_common\PhotographablePreview.cs" />
+    <Compile Include="src\gui_common\thriveopedia\Thriveopedia.cs" />
```

Comment:
I think as the thriveopedia is a pretty big feature by itself (maybe even as big as some editors) that it would kind of make sense for it to be not under gui_common folder, instead being directly under "src", as it's not really a common GUI component building block. And the fossilization folder could also then put under the thriveopedia folder to group those interrelated functionalities together.